release_notes/utils.py

```diff
@@ -272,3 +276,30 @@ def request_pull_requests_from_api(
                            f'{pending.keys()} is/are either not closed or cannot be found')
 
     return result
+
+
+def release_notes_files_into_docs(
+    repo_dir: str,
+    rel_path: str='.ocm/release-notes',
+) -> list[rnm.ReleaseNotesDoc]:
+    docs_dir = os.path.join(
+        repo_dir,
+        rel_path,
+    )
+    return read_release_notes_from_dir(docs_dir)
```

Comment:
since this function, again, basically just wraps rrnfd, why not drop it.

we could simply keep accepting `release_notes_dir` as single parameter, and default it to `.ocm/release-notes`; I think there is limited value from accepting two parameters. It should be okay to make it the caller's responsibility to tell us the directory (-> separation of concern)